packages/api/config/lambdas.yml

```diff
@@ -178,3 +178,17 @@ CreateReconciliationReport:
 BulkDelete:
   handler: index.bulkDeleteLambda
   source: 'node_modules/@cumulus/api/dist/'
+
+KinesisRuleInput:
```

Comment:
I'm not happy with these lambda names.  Suggestions?

Reply:
I've thought about this a bit more and I think having "logger" somewhere within the lambda name would make things more easy to intuit. Something like `KinesisRuleInputLogger`?

Reply:
@ifestus that can work, KinesisInboundEventLogger/KinesisOutboundEventLogger?

Reply:
@ifestus  8bd990c